archlinuxcn/devtools-archlinuxcn/PKGBUILD

```diff
@@ -15,6 +15,9 @@ optdepends=('btrfs-progs: btrfs support')
 provides=("devtools=$pkgver-$pkgrel")
 conflicts=("devtools")
 source=(https://sources.archlinux.org/other/${_pkgname}/${_pkgname}-${pkgver}.tar.gz{,.sig}
+        "01-nosetarch.patch::https://github.com/archlinux/devtools/commit/61f67fd05192a21c03a015712b0eae749ce81ccf.patch"
+        "02-nosetarch.patch::https://github.com/archlinux/devtools/commit/264b9a915ca25793d71de58a304a62ec20970e08.patch"
```

Comment:
Ditto. And it is better to merge those two patches. (spliting them is easier for reviewing, but not a good idea for PKGBUILD).